### PR TITLE
added info on dataprocessing

### PR DESCRIPTION
**Pull request by TheTechArch**

### CODE REVIEW

File: content/altinn-studio/app-creation/logic/calculation/_index.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ toc: true
 ---
 
 
-## Kalkulering
+**Kalkulering er fra 4.7.0 erstattet av dataprosessering. Se [her](./../dataprocessing/_index.md**
```
Comment (altinnadmin) [inline content/altinn-studio/app-creation/logic/calculation/_index.md:8]: Ser ikke ut som en gyldig lenke.

File: content/altinn-studio/app-creation/logic/dataprocessing/_index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Eksempel på kode som erstatter en gitt verdi (`12345678`) med en annen verdi (`22222222`) i et gitt felt vises under:
+
+```C# {hl_lines=[16,22]}
+public async Task<bool> ProcessDataWrite(Instance instance, Guid? dataId, object data)
+{
+    if (data.GetType() == typeof(Skjema))
+    {
+        // Cast the data object to model type to access all fields
+        Skjema model = (Skjema)data;
```
Comment (altinnadmin) [inline content/altinn-studio/app-creation/logic/dataprocessing/_index.md:34]: Better to not use an OR datamodel in this example? Is very verbose and really an anti-pattern when it comes to data modelling.

File: content/altinn-studio/app-creation/logic/dataprocessing/_index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        // Get the existing value of a specified field, if it exists
+        string tlf = 
+            model?
+            .OpplysningerOmArbeidstakerengrp8819?
+            .OpplysningerOmArbeidstakerengrp8855?
```
Comment (altinnadmin) [inline content/altinn-studio/app-creation/logic/dataprocessing/_index.md:40]: Verbose indeed... :)

**Review by altinnadmin — APPROVED** [review]
Nice, a couple of comments to fix 👍

File: content/altinn-studio/app-creation/logic/calculation/_index.md (hunk 1 above)
Comment (TheTechArch) [reply content/altinn-studio/app-creation/logic/calculation/_index.md]: fixed

File: content/altinn-studio/app-creation/logic/dataprocessing/_index.md (hunk 3 above)
Comment (TheTechArch) [reply content/altinn-studio/app-creation/logic/dataprocessing/_index.md]: endret til kode fra UDI app

File: content/altinn-studio/app-creation/logic/dataprocessing/_index.md (hunk 2 above)
Comment (TheTechArch) [reply content/altinn-studio/app-creation/logic/dataprocessing/_index.md]: tok kode fra UDI app

**Pull request merged**